website/docs/docs/deploy/source-freshness.md

```diff
@@ -12,7 +12,14 @@ dbt Cloud provides a helpful interface around dbt's [source data freshness](/doc
 
 ### Enabling source freshness snapshots
 
-First, make sure to configure your sources to [snapshot freshness information](/docs/build/sources#snapshotting-source-data-freshness).
+`dbt build` does _not_ include source freshness checks when it builds and tests resources in your DAG. As such, here's a common pattern for defining jobs:
+- `dbt build` as the run step
+- checkbox for [generating docs](/docs/collaborate/build-and-view-your-docs#set-up-a-documentation-job)
+- checkbox for [source freshness](#checkbox)
```

Comment:
Might be good to give a description of what this pattern does

Reply:
great flag @graciegoheen - is this what you meant about clarification?